### DISCUSSION TITLE
CONVERSATIONS ABOUT THE ACCESSIBILITY

### DISCUSSION THREAD

**Discussion by vigneshraaj0** [community/community#123983] [Accessibility]
### Select Topic Area

Question

### Body

Accessibility Features: GitHub has been working on making its platform more accessible to users with disabilities. This includes features like keyboard navigation, screen reader compatibility, and ensuring proper contrast for users with low vision.

Accessibility Guidelines: GitHub provides guidelines for making repositories accessible. This includes using semantic HTML, providing alternative text for images, ensuring proper focus management, and following best practices for web accessibility.

Community Efforts: GitHub encourages community contributions to improve accessibility. Users can submit accessibility-related issues and pull requests to help identify and fix accessibility issues across the platform.

Documentation: GitHub provides documentation on accessibility features and best practices for developers. This includes guidance on designing accessible user interfaces, creating accessible documentation, and testing for accessibility.

Ongoing Improvements: GitHub continues to work on improving accessibility across its products. This includes addressing feedback from users with disabilities, conducting accessibility audits, and implementing new features to enhance accessibility.

**Comment by vigneshraaj00** [comment]
Response #2

**Comment by ebndev** [comment]
Hi @vigneshraaj0 & @vigneshraaj00, thanks for being a part of the GitHub Community!

Please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
